### PR TITLE
Applied mypy + flake8 + ISort for all files

### PR DESCRIPTION
**Pull request by RukhovichIV**
A first part of the refactoring

### CODE REVIEW

**Review by PetrovKP — APPROVED** [review]

File: datasets/load_datasets.py [hunk 1]
```diff
@@ -41,7 +40,7 @@ def try_load_dataset(dataset_name, output_directory):
     if dataset_name in dataset_loaders.keys():
         try:
             return dataset_loaders[dataset_name](output_directory)
-        except:
+        except BaseException:
             logging.warning("Internal error loading dataset")
```
Comment (Alexsandruss) [inline datasets/load_datasets.py:44]: In this handling `except BaseException` is not better than `except`. Add traceback in warning or print exception name at least:
```
except BaseException as e:
    logging.warning(f"Internal error loading dataset: {e}")
```

File: sklearn_bench/df_clsf.py [hunk 2]
```diff
@@ -46,7 +43,8 @@
 
 params = bench.parse_args(parser)
 
-from sklearn.ensemble import RandomForestClassifier
+if not params.no_intel_optimized:
+    from sklearn.ensemble import RandomForestClassifier
```
Comment (Alexsandruss) [inline sklearn_bench/df_clsf.py:47]: Where is RandomForestClassifier if params.no_intel_optimized=True? Is benchmark predicted to be failed in this case?

File: sklearn_bench/knn_clsf.py [hunk 3]
```diff
@@ -39,7 +37,8 @@
                     help='Distance metric to use')
 params = bench.parse_args(parser)
 
-from sklearn.neighbors import KNeighborsClassifier
+if not params.no_intel_optimized:
+    from sklearn.neighbors import KNeighborsClassifier
```
Comment (Alexsandruss) [inline sklearn_bench/knn_clsf.py:41]: It is not clear why estimators are imported only in one case

**Review by Alexsandruss — CHANGES_REQUESTED** [review]

File: sklearn_bench/df_clsf.py (hunk 2 above)
Comment (RukhovichIV) [reply sklearn_bench/df_clsf.py:47]: My bad. I just wanted to separate this import from others, so we can ISort them

File: datasets/load_datasets.py (hunk 1 above)
Comment (RukhovichIV) [reply datasets/load_datasets.py]: done

File: sklearn_bench/df_clsf.py (hunk 2 above)
Comment (RukhovichIV) [reply sklearn_bench/df_clsf.py]: changed the behaviour. It should work good now

File: sklearn_bench/knn_clsf.py (hunk 3 above)
Comment (RukhovichIV) [reply sklearn_bench/knn_clsf.py]: Also done

**Review by Alexsandruss — APPROVED** [review]

**Pull request merged**